scripts/publish_prod.sh

```diff
@@ -37,3 +37,6 @@ git push origin "refs/tags/v$PACKAGE_VERSION"
 echo 'Publishing Lambda Layer'
 ./scripts/build_layers.sh
 ./scripts/publish_layers.sh
+
+echo 'Checking Regions'
+./scripts/check_regions.sh
```

Comment:
We should run check_regions before publishing, that way we won't accidentally publish a new version 1 of the layer in the new region, which I think would break the add_region script.

Reply:
✅